### PR TITLE
[TR] PIM-4458: Fix name display in pinbar for product edit pages

### PR DESCRIPTION
**Pull request by phaseinducer**
https://akeneo.atlassian.net/browse/PIM-4458

| Q                 | A
| ----------------- | ---
| Added Specs       | n/a
| Added Behats      | ok
| Travis CI is ok   | yes
| Changelog updated | yes

### CODE REVIEW

**Comment by grena** [comment]
Nice one, can't wait to drop this \o/
GTM

File: features/Context/NavigationContext.php [hunk 1]
```diff
@@ -688,6 +688,32 @@ public function iRefreshCurrentPage()
...
+    {
+        $pinButton = $this->spin(function () {
+            return $this->getCurrentPage()->find('css', '.minimize-button');
+        }, 10);
+
+        $pinButton->click();
+    }
+
+    /**
+     * @When /^I click on the pinned item "(.*)"$/
```
Comment (jmleroux) [inline features/Context/NavigationContext.php]: `([^"]+)`

File: features/navigation/pin_pages.feature [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    Given a "default" catalog configuration
+    And the following products:
+      | sku       |
+      | pineapple |
+      | potatoe   |
+    And I am logged in as "Mary"
+
+  Scenario: Add pages to the pinbar
+    Given I am on the "pineapple" product page
+    Then the product SKU should be "pineapple"
```
Comment (jmleroux) [inline features/navigation/pin_pages.feature]: Not usefull imho, this is not the object of the test

**Comment by jmleroux** [comment]
GTM

File: features/navigation/pin_pages.feature (hunk 2 above)
Comment (phaseinducer) [inline features/navigation/pin_pages.feature]: This step is used to wait for the product to be fetched.

**Pull request merged**